extension-requests/local-utils.js

```diff
@@ -236,10 +236,31 @@ function dateTimeString(milliseconds) {
   ).padStart(2, '0')}:${String(date.getMinutes()).padStart(2, '0')}`;
 }
 
+/**
+  Generates a formatted date string from milliseconds.*
+  @param {number} milliseconds - The number of milliseconds since January 1, 1970 00:00:00 UTC.
+  @returns {string} The formatted date string in the format 'YYYY-MM-DD'.
+  
+*/
+function dateString(milliseconds) {
+  const date = new Date(milliseconds);
+  return `${date.getFullYear()}-${String(date.getMonth() + 1).padStart(
+    2,
+    '0',
+  )}-${String(date.getDate()).padStart(2, '0')}`;
```

Comment:
what if the month is already two digit?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart
![Screenshot 2024-01-02 at 7 45 25 PM](https://github.com/Real-Dev-Squad/website-dashboard/assets/98796547/a6cd2085-7a1d-45ea-bdc2-a6bb2d823d01)